src/io/flutter/FlutterUtils.java

```diff
@@ -30,10 +32,15 @@ public static void invokeAndWait(@NotNull Runnable runnable) throws ProcessCance
 
   @SuppressWarnings("BooleanMethodIsAlwaysInverted")
   public static boolean isFlutteryFile(@NotNull VirtualFile file) {
-    final String fileName = file.getName();
-    return file.getFileType() == DartFileType.INSTANCE ||
-           fileName.equals(FlutterConstants.FLUTTER_YAML) ||
-           fileName.equals(FlutterConstants.PUBSPEC_YAML);
+    return isDartFile(file) || isPubspecFile(file);
```

Comment:
We _may_ want to keep `flutter.yaml` in the check for a release or two (to identify the project as a flutter project), in order to give people a few weeks to migrate.